SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/Digital/PPMU.cs

```diff
@@ -106,6 +106,35 @@ public static void ForceVoltage(
             });
         }
 
+        /// <summary>
+        /// Forces voltage on the target pin(s).
+        /// </summary>
+        /// <param name="sessionsBundle">The <see cref="DigitalSessionsBundle"/> object.</param>
+        /// <param name="settings">The settings to use.</param>
+        public static void ForceVoltage(this DigitalSessionsBundle sessionsBundle, PPMUSettings settings)
+        {
+            sessionsBundle.Do(sessionInfo =>
+            {
+                settings.OutputFunction = PpmuOutputFunction.DCVoltage;
+                sessionInfo.Session.Force(sessionInfo.PinSetString, settings);
+            });
+        }
+
+        /// <summary>
+        /// Forces voltage on the target pin(s).
```

Comment:
Suggest changing to: "Forces voltage using specified site-unique ppmu settings."